tuned/plugins/plugin_sysctl.py

```diff
@@ -51,8 +51,8 @@ def _instance_apply_static(self, instance):
 		for option, value in list(instance._sysctl.items()):
 			original_value = _read_sysctl(option)
 			if original_value is None:
-				log.error("sysctl option %s will not be set, failed to read the original value."
-						% option)
+				log.warn("sysctl option %s will not be set, failed to read the original value."
+						 % option)
```

Comment:
We'll probably solve this another way - we'll add a way to differentiate sysctl settings that are required vs. not required. Failures to set required settings will result in error messages, whereas failures to set non-required settings will result in warning messages (this was requested in https://bugzilla.redhat.com/show_bug.cgi?id=1666562#c4). Until that's implemented, I think it's safer to keep printing error messages.

I have a work-in-progress solution to that here: https://github.com/olysonek/tuned/commit/adc926b756761e0e6784bc51b017cb1112dffb63

You can override the Tuned profile and drop the missing sysctl settings to make the error message go away.